phpunit.xml.dist

```diff
@@ -53,6 +53,7 @@
     <php>
         <ini name="zend.enable_gc" value="0"/>
         <ini name="memory_limit" value="1G"/>
+        <env name="FAST_LINT_TEST_CASES" value="0"/>
```

Comment:
What is the purpose of this variable? I mean: if it only changes the time required to lint files, shouldn't we always enable it? Or is there any trade-off when doing so?

Reply:
All these changes are from another PR. Please review that one. I will rebase this after that has merged. ;)

NB The reason for fast lint was because the tests took over an hour otherwise after the full linting was added. The reason for the full linting is explained in my other PR.